### PR TITLE
Fixing small typos on functional_connectomes.rst

### PR DESCRIPTION
**Pull request by Naereen**
Fixing small typos on doc/connectivity/functional_connectomes.rst.

### CODE REVIEW

File: doc/connectivity/functional_connectomes.rst [hunk 1]
```diff
@@ -105,38 +105,40 @@ regions, regressing out noise sources is indeed very important
...
-----------------------
+---------------------
 
 The definition of regions as by a continuous probability map captures
 better our imperfect knowledge of boundaries in brain images (notably
 because of inter-subject registration errors). One example of such an
 atlas well suited to resting-state data analysis is the `MSDL atlas
-<https://team.inria.fr/parietal/research/spatial_patterns/spatial-patterns-in-resting-state/>`_ (:func:`nilearn.datasets.fetch_atlas_msdl`).
+<https://team.inria.fr/parietal/research/spatial_patterns/spatial-patterns-in-resting-state/>`_
+(:func:`nilearn.datasets.fetch_atlas_msdl`).
```
Comment (aabadie) [inline doc/connectivity/functional_connectomes.rst]: Any reason for splitting this line ?

File: doc/connectivity/functional_connectomes.rst [hunk 2]
```diff
@@ -105,38 +105,40 @@ regions, regressing out noise sources is indeed very important
...
 
 Probabilistic atlases are represented as a set of continuous maps, in a
 4D nifti image. Visualization the atlas thus requires to visualize each
 of these maps, which requires accessing them with
-:func:`nilearn.image.index_img` (see the :ref:`corresponding example <sphx_glr_auto_examples_01_plotting_plot_overlay.py>`).
+:func:`nilearn.image.index_img` (see the :ref:`corresponding example
+<sphx_glr_auto_examples_01_plotting_plot_overlay.py>`).
```
Comment (aabadie) [inline doc/connectivity/functional_connectomes.rst]: Same question here. (just curious)

Comment (Naereen) [inline doc/connectivity/functional_connectomes.rst]: All the rST files in this repo (and others) try to keep lines below 80 characters, these harmless couple of line splits were done just to keep a more consistent line width style.

Comment (aabadie) [inline doc/connectivity/functional_connectomes.rst]: Ok, I was just wondering if it won't change the final display result (let's wait for circle ci to finish).

Comment (Naereen) [inline doc/connectivity/functional_connectomes.rst]: No, sphinx, as latex, usually don't care for line splitting (except if it between markup). In this case, for links, it's ok to split the line before the `<...>_` part.

File: doc/connectivity/functional_connectomes.rst [hunk 3]
```diff
@@ -231,9 +233,8 @@ only direct connections.
 
 .. topic:: **References**
 
-  * `Zalesky NeuroImage 2012 "On the use of correlation as a measure of
-    network connectivity" <http://www.sciencedirect.com/science/article/pii/S1053811912001784>`_
+  * `Zalesky, NeuroImage 2012, "On the use of correlation as a measure of
```
Comment (banilo) [inline doc/connectivity/functional_connectomes.rst]: adding "et al." here and below might give it a more sciency spin

Comment (Naereen) [inline doc/connectivity/functional_connectomes.rst]: OK, done, [3ee1c69](https://github.com/Naereen/nilearn/commit/3ee1c693563f5c8babf26b8ec809914bd4d196b8).

**Comment by Naereen** [comment]
The checks passed, and I added your requests. Can you merge it?

**Comment by bthirion** [comment]
LGTM.

**Comment by aabadie** [comment]
LGTM. Merging, thanks !

**Comment by Naereen** [comment]
Thanks!

**Pull request merged**